flit_core/flit_core/common.py

```diff
@@ -18,8 +18,11 @@ class Module:
     in_namespace_package = False
     namespace_package_name = None
 
-    def __init__(self, name, directory=Path()):
+    is_stub_pkg: bool
+
+    def __init__(self, name: str, directory=Path()):
         self.name = name
+        self.is_stub_pkg = name.endswith('-stub')
```

Comment:
```suggestion
    def __init__(self, name: str, directory=Path()):
        self.name = name
        self.is_stub_pkg = name.endswith('-stub')
```

No need to add the explicit annotation here. Type checkers will be able to infer it.

Reply:
at least pyright can't? I didn't get type info in vscode

Reply:
> at least pyright can't? I didn't get type info in vscode

It works once `name` is annotated as `str`.

<img width="618" alt="Screenshot 2025-05-07 at 12 41 40" src="https://github.com/user-attachments/assets/264bd6c3-3327-4efc-b2d0-0a7fe90f11fc" />